docs/pages/elevation.js

```diff
@@ -0,0 +1,166 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Flex, Text } from 'gestalt';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import Page from '../components/Page.js';
+
+export default function ColorUsagePage(): Node {
+  return (
+    <Page title="Elevation">
```

Comment:
This doesn't include any information about elevation vis-a-vis dark mode. If that's coming in the future, should we add a note indicating that?

Reply:
Given that we don't already support dark mode, I think we can leave that off for now. Plus dark mode is being added next sprint